<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MeasurementWidget</class>
 <widget class="QWidget" name="MeasurementWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>763</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>快速启动模式</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0" colspan="2">
          <widget class="QRadioButton" name="radioButton">
           <property name="text">
            <string>立即开始测量，持续 1 小时</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QRadioButton" name="radioButton_2">
           <property name="text">
            <string>立即开始测量，持续 3 小时</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="2">
          <widget class="QRadioButton" name="radioButton_3">
           <property name="text">
            <string>立即开始测量，持续 6 小时</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0" colspan="3">
          <widget class="QRadioButton" name="radioButton_4">
           <property name="text">
            <string>立即开始测量，持续直到存储器满</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QRadioButton" name="radioButton_5">
           <property name="text">
            <string>指定测量时间：</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>开始时间</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QDateTimeEdit" name="dateTimeEdit_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="displayFormat">
            <string>yyyy-MM-dd HH:mm:ss</string>
           </property>
           <property name="calendarPopup">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>结束时间</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="5" column="2">
          <widget class="QDateTimeEdit" name="dateTimeEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="displayFormat">
            <string>yyyy-MM-dd HH:mm:ss</string>
           </property>
           <property name="calendarPopup">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>记录间隔：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
          <property name="suffix">
           <string> 秒</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="6" column="0">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
        <property name="title">
         <string>串口操作</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0" colspan="2">
          <widget class="QLabel" name="label_4">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string>串口图标</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QComboBox" name="comboBox"/>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="pushButton_4">
           <property name="text">
            <string>刷新串口</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="pushButton_5">
           <property name="text">
            <string>关闭串口</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="5" column="0">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="0">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>激活设备</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_3">
          <property name="text">
           <string>停止设备</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="helpButton">
          <property name="text">
           <string>?</string>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <property name="helpButton" stdset="0">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>149</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="2" rowspan="2">
    <widget class="QGroupBox" name="devicesGroupBox">
     <property name="title">
      <string>设备</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="1" column="0">
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>读取设备</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="refreshButton">
        <property name="text">
         <string>刷新设备</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QScrollArea" name="devicesScrollArea">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>176</width>
           <height>505</height>
          </rect>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
